OutSystems/outsystems-maps
# ROU-4763: Add ability to to GoogleMaps to add localization by setting the language and region

This PR is for adding the **Localization** feature to `GoogleMaps`, `StaticMap` and `SearchPlaces`  by setting the **language** and **region** through `OptionalConfigs` on each component.

### What was happening
* The user to apply a localization on a google maps, he need to add it on APIKey value as workaround. With this, the google maps will have the language and region applied.

### What was done
* Add the Localization parameter into OptionalConfigs on GoogleMaps, StaticMap and SearchPlaces
* Review the TypeScript to add the localization (language and region) into consideration on loaded script by google
![image](https://github.com/OutSystems/outsystems-maps/assets/25321845/ada17ab3-9f9a-43cd-b954-bf55e95083ab)
![image](https://github.com/OutSystems/outsystems-maps/assets/25321845/a15ffc30-01a4-4273-9131-e1be0b207d20)

### Test Steps
1. Open each Sample Page
2. **Expected:** Map will rendered in Japan language _(On StaticMap, you can change the language and toogle the map visibility to apply the changes.)_

### Checklist
